tensorboard/webapp/widgets/line_chart_v2/sub_view/line_chart_axis_view.ts

```diff
@@ -101,4 +110,34 @@ export class LineChartAxisComponent {
       ? LineChartAxisComponent.TEXT_PADDING
       : this.getDomPos(tick);
   }
+
+  keydownPreventClose(event: KeyboardEvent) {
+    if (event.key !== 'Escape') {
+      event.stopPropagation();
```

Comment:
Can we add a TODO / comment explaining why we need to stopPropagation to prevent MatMenu's default behavior?  I think it's not super obvious without having manually playing with this feature.